docs/source/developing/ci.adoc

```diff
@@ -10,21 +10,45 @@ toc::[]
 [[ci-overview]]
 == Overview
 
-The following sections describes the various CI setups Minishift utilizes.
+Minishift uses the following CI build servers: CentOS, AppVeyor, Circle, and Travis. These servers build incoming pull requests and provide the status of the builds to ensure continuous integration. Except for Travis CI, they also archive build artifacts of successful builds.
 
-[[ci-setup]]
+image::<CI_Servers>[CI Servers]
```

Comment:
Will that work when the docs are integrated in docs.openshift.org. I don't think so. I don't even think that we include images atm into the tarball. So we might need to adjust the docs build here as well.

@Preeticp Have you tried building the full site - https://github.com/minishift/minishift/blob/master/docs/source/developing/writing-docs.adoc#integration-with-docs-openshift-org?

Also according to the guidelines - https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#images - images should be relative to the topic.